lib/src/widgets/switch/ds_switch_tile.widget.dart

```diff
@@ -0,0 +1,75 @@
+import 'package:blip_ds/blip_ds.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+class DsSwitchTile extends StatelessWidget {
```

Comment:
uses DSSwitchTile instead of DsSwitchTile

Reply:
Done